### PR TITLE
Update Wazuh v3.13.0 release notes

### LINKED ISSUE #2592: Create release notes for 3.13
Hello team!

This issue aims to create the release notes for Wazuh 3.13.

Regards,

David

### PR DESCRIPTION
**Pull request by davidfmiranda**
Hello team!

This PR closes #2592 

## Description

I've updated the existing release notes with the latest changes made to Wazuh 3.13.0

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

Regards,

David

### CODE REVIEW

File: source/release-notes/release_3_13_0.rst [hunk 1]
```diff
@@ -23,34 +24,34 @@ Wazuh core
 - Added a timeout to the updates of the vulnerability detector's feeds to prevent hangings.
 - Added option for the JSON decoder to choose the treatment of array structures.
 - Added ``mode`` value (real-time, Who-data, or scheduled) as a dynamic field in FIM alerts. 
-- Added a configurable maximum limit of files to be monitored by FIM.
-- New integration for pull logs from Google Cloud Pub/Sub.
-- Added support for MITRE ATT&CK knowledge base.
+- Added a field to configure the maximum files to be monitored by the FIM.
```
Comment (alberpilot) [inline source/release-notes/release_3_13_0.rst:27]: By the FIM? FIM module sounds better, right?

File: source/release-notes/release_3_13_0.rst [hunk 2]
```diff
@@ -23,34 +24,34 @@ Wazuh core
...
+- Added a module to integrate Google Cloud Pub/Sub.
+- Added support for mapping Wazuh rules with MITRE ATT&CK knowledge base.
 - Added new configuration block for Windows Defender EventChannel.
```
Comment (alberpilot) [inline source/release-notes/release_3_13_0.rst:30]: Please remove this, it was finally removed (Windows Defender configuration block).

File: source/release-notes/release_3_13_0.rst [hunk 3]
```diff
@@ -23,34 +24,34 @@ Wazuh core
...
 
 **Other fixes and improvements**
 
 - Fixed a bug that occasionally, kept the memory reserved when deleting monitored directories in FIM.
-- Freed Inotify watches moving directories in the real-time mode of FIM.
-- Fixed an error that caused the alerts' deletion with a wrong path in Who-data mode.
-- Fixed generating alerts in Who-data mode when moving directories to the folder being monitored in Windows.
+- Fixed an issue that prevented Inotify to be liberated when modifying directories in real-time mode of FIM.
+- Fixed an error that caused the alerts deletion with a wrong path in Who-data mode.
+- Fixed and issue that did not generate alerts in Who-data mode when a subdirectory was added to the monitored directory in Windows.
```
Comment (alberpilot) [inline source/release-notes/release_3_13_0.rst:52]: Fixed and issue? or an issue?

File: source/release-notes/release_3_13_0.rst (hunk 1 above)
Comment (davidfmiranda) [reply source/release-notes/release_3_13_0.rst:27]: Agree

File: source/release-notes/release_3_13_0.rst (hunk 2 above)
Comment (davidfmiranda) [reply source/release-notes/release_3_13_0.rst:30]: Done

File: source/release-notes/release_3_13_0.rst (hunk 3 above)
Comment (davidfmiranda) [reply source/release-notes/release_3_13_0.rst:52]: Fixed

**Pull request merged**